### DISCUSSION TITLE
Transfer domains out of Vercel - stuck?

### DISCUSSION THREAD

**Discussion by idevelop** [vercel/community#3581] [Help] [answered]
### Question

Hi, on August 3rd I initiated 2 domain transfers out from Vercel into Cloudflare, for the domains crongpt.dev and coachgpt.app. These domains were previously transferred in July from Google Domains into Vercel. They still haven't moved over to Cloudflare, with the status there being "Approve transfer at Unknown (GoogleRegistry-tucows)".

I don't see any approval flow in Vercel.

Can you please advise?

Thanks!

**Comment by smaeda-ks** [comment]
It looks like they are now moved to Cloudflare. The process usually takes up to 5 days, but Vercel support can also manually approve requests to make it faster (i.e., if you can submit a support ticket as a Pro plan user).

**Reply by idevelop** [reply]
@smaeda-ks where are you seeing them transferred? In my Cloudflare domains page I still see it as "Approve transfer at Unknown (GoogleRegistry-tucows)".

<img width="739" alt="Screenshot 2023-08-17 at 12 13 40" src="https://github.com/vercel/community/assets/215035/82cce442-6691-4288-b13b-abe31e271d9d">

**Reply by idevelop** [reply]
Also still seeing both domains in my Vercel Domains dashboard, appearing as registered with Vercel.

**Comment by amyegan** [comment, marked as answer]
Hey @idevelop. I think the domains were transferred too recently. You need to wait at least 60 days after a transfer before you can start a new transfer. 

> [ICANN](https://www.icann.org/) forces domain registrars to wait 60 days:
> - between transfers
> - between a new registration and a subsequent transfer
> 
> If you transfer before this time, the transfer will fail. Besides this restriction, some DNS providers may further restrict domain transferring by default as a security measure, unless the owner explicitly turns off their protection setting. Please refer to the DNS provider's documentation for more details.
>
> —  https://vercel.com/docs/concepts/projects/domains/troubleshooting#unable-to-transfer-in-a-domain
